src/configure/configure.ts

```diff
@@ -176,17 +175,19 @@ class Orchestrator {
 
             try {
                 if (!resourceNode) {
-                    this.context['rightClickScenario'] = false;
+                    this.context['resourceId'] = '';
+                    this.context['isResourceAlreadySelected'] = false;
                     await this.getAzureSubscription();
                     await this.getAzureResource(this.getSelectedPipelineTargetType());
                 }
                 else {
-                    this.context['rightClickScenario'] = true;
+                    this.context['isResourceAlreadySelected'] = true;
                 }
 
                 if (this.inputs.targetResource.resource && this.inputs.targetResource.resource.id) {
                     this.context['resourceId'] = this.inputs.targetResource.resource.id;
```

Comment:
should we not do `this.context['isResourceAlreadySelected'] = true;` here rather than in L184. This way there is no inconsistency. We should not have isResourceAlreadySelected = true but resourceId as empty.

Reply:
Thinking more about it, this looks okay

Reply:
Or better move setting these context items to `analyzeRepo` function. Overall concern is that same context is being set at multiple places making it difficult to understand.